features/core/onboarding.feature

```diff
@@ -125,3 +125,102 @@ Scenario: Registering a new company account on evan
   #   And The value of the Input field with label "Website" should be "https://evan.network"
+
+Scenario: Creating a new bycicle twin using onboarding
+
+  Given I go to the evan.network startpage
+    And I open the path "dashboard.vue.evan/onboarding.vue.evan/twin-sign-up"
+    Then I want to see a text including "Create your Digital Twin"
+    And I want to see a text including "Choose a twin type and give your new twin meaningful name"
+    And the button "Continue" should be "disabled"
+
+  When I set Input field with label "Type" to "bycicle"
+    When I set Input field with label "Name" to "Onboarding Automated Twin"
+    And I set Input field with label "Description" to "Enim fugiat magna sunt consectetur pariatur irure elit laborum ea tempor."
+    Then the button "Continue" should be "enabled"
+
+  When I click on button "Continue"
+    Then the button "Continue" should be "disabled"
+   # Then I want to see a text including "Describe your bycicle by it's specifications"
+
+  When I set Input field with label "Type" to "mountain bike"
+    And I set Input field with label "Color" to "red"
+    And I set Input field with label "Wheel size" to "29"
+    And I set Input field with label "Gender" to "male"
+    And I set Input field with label "Manufacturer" to "Ghost"
+    Then the button "Continue" should be "enabled"
+
+  When I click on button "Continue"
+    Then the button "Continue" should be "disabled"
+    And I want to see a text including "Complete history of maintenance logs."
+    And I want to see a text including "Date"
+    And I want to see a text including "Description"
+    And I want to see a text including "Processor"
+
+  When I set Input field with id "date0" to "01.01.2018"
+    And I set Input field with id "date1" to "02.03.2019"
+    And I set Input field with id "date2" to "30.11.2019"
+
+    And I set Input field with id "description0" to "service start"
+    And I set Input field with id "description1" to "service end"
+    And I set Input field with id "description2" to "rent start"
+
+    And I set Input field with id "processor0" to "User 1"
+    And I set Input field with id "processor1" to "User 2"
+    And I set Input field with id "processor2" to "User 3"
+
+    Then the button "Continue" should be "enabled"
+
+  When I click on button "Continue"
+    Then the button "Create Account" should be "disabled"
+    And I want to see a text including "For your access to your Digital Twin, we need an alias and a password."
+    And I set Input field with label "Account name" to "Automated Test Account"
+    And I set Input field with label "Password" to "Test1234"
+    And I set Input field with label "Retype password" to "Test1234"
+    And I click the element with id "evan-recaptcha"
+    And I want to click on vue checkbox control with id "termsAccepted"
+    And I wait for 3 seconds
+    Then the button "Create Account" should be "enabled"
+
+  # TODO: wait until profile was created
```

Comment:
coo, if the commented code is used soon. 
Otherwise a small reminder:  
https://kentcdodds.com/blog/please-dont-commit-commented-out-code

:)